src/publisher/tests/test_api_v2_views.py

```diff
@@ -310,30 +316,58 @@ def setUp(self):
         self.assertTrue(fragments.merge_if_valid(self.av))
 
         self.c = Client()
+        self.ac = Client(**{
+            mware.CGROUPS: 'admin',
+        })
 
     def test_add_fragment(self):
         "a POST request can be sent that adds an article fragment"
         key = 'test-frag'
         url = reverse('v2:article-fragment', kwargs={'art_id': self.msid, 'fragment_id': key})
-        fragment = {'title': 'pants-party'}
```

Comment:
you don't like the `pants-party` ? heh. an old boss used that as his 'foobar'. I have no idea what it means, but it's something that keeps on being funny to me